ReSharper.FSharp/test/data/features/daemon/typeHints/Signatures - Top level 01.fs.gold

```diff
@@ -41,34 +41,34 @@ let g||(15) = fun x -> ()
 
 type A(x||(16)) =
     do
-        let x||(17) = 3
```

Comment:
Fixed the case when hints showed for some local bindings even if it was disabled